src/app/components/Statistics/CO2ConsumptionCharts/CountriesBarMeter.tsx

```diff
@@ -26,7 +26,7 @@ const CountriesBarMeter = ({ darkMode, titleText, dataSeries }) => {
         },
         title: {
             align: 'left',
-            text: titleText,
+            text: '',
```

Comment:
can you remove unused titleText from props

Reply:
When I delete this property, "Chart Title" appears by default but we already have the title of the chart on the Card.
Then add a "display: none" so that it does not appear and an empty title does not remain in the code, but it does appear empty in the graphic